web/src/js/gcping.js

```diff
@@ -35,56 +35,62 @@ const GLOBAL_REGION_KEY = "global",
  *  }
  * }
  */
-let regions = {},
-  results = [],
-  pingTestStatus = PING_TEST_RUNNING_STATUS,
-  fastestRegionVisible = false,
-  globalRegionProxy = '';
+const regions = {};
+const results = [];
+let pingTestStatus = PING_TEST_RUNNING_STATUS;
+let fastestRegionVisible = false;
+let globalRegionProxy = "";
 
 /**
  * Fetches the endpoints for different Cloud Run regions.
  * We will later send a request to these endpoints and measure the latency.
  */
-function getEndpoints(){
-  fetch("/api/endpoints").then(function(resp) {
-    return resp.json();
-  }).then(function(endpoints) {
-    for (zone of Object.values(endpoints)) {
-      let gcpZone = {
-        key: zone.Region,
-        label: zone.RegionName,
-        pingUrl: zone.URL + "/api/ping",
-        latencies: [],
-        median: ''
-      };
-
-      regions[gcpZone.key] = gcpZone;
-    }
+function getEndpoints() {
+  fetch("/api/endpoints")
```

Comment:
Not sure how the `quotes` rule got removed here, perhaps part of the react plugin we removed? 🤔